### ISSUE TITLE
[TF 2.0 API Docs] `tf.keras.callbacks.LearningRateScheduler`

### ISSUE THREAD

**Issue by 8bitmp3** [tensorflow/tensorflow#30002]
TensorFlow version: 2.0 (beta1)

#### Doc links:
https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/callbacks/LearningRateScheduler
https://github.com/tensorflow/tensorflow/blob/r2.0/tensorflow/python/keras/callbacks.py

#### Description:
The definition should be expanded for clarity and better user experience. Instead of a one short sentence - "Learning rate scheduler." - maybe try:
```
Learning rate scheduler which can be used to adjust the learning rate over time. Any function can be defined, such as a decay function that states which decay rate to use after a certain number of epochs or batches. Then this custom function can be passed as the `schedule` parameter in the `LearningRateSchuler` callback. 
```
Alternatively, a description from the "Distributed Training with Keras" [tutorial](https://www.tensorflow.org/beta/tutorials/distribute/keras), linked in the API doc, can also be used:
```
Learning Rate Scheduler: Using this callback, you can schedule the learning rate to change after every epoch/batch.
```
#### Example:
The word "Example" is missing before the example.

#### Raises/Returns:
Should be added.

For improvement suggestions of other `tf.keras.callbacks` classes see #29958

**Comment by 8bitmp3** [comment]
https://github.com/tensorflow/tensorflow/commit/089f8e2c0edb6284e5c3a3ba1042f410ac41b6e6 @achandraa and @jvishnuvardhan - did a bit of work with the description and a few minor changes in the style of the `ReduceLROnPlateau` callback

**Comment by jvishnuvardhan** [comment]
@8bitmp3 Are you raising a PR? Thanks!

**Comment by jvishnuvardhan** [comment]
I am closing the issue as it was stale issue. Please feel free to reopen when you have time to update the information. Thanks!
